doc/api/test.md

```diff
@@ -1620,6 +1675,51 @@ describe('tests', async () => {
 });
 ```
 
+## `config`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1.0 - Early development
+
+An object whose methods are used to configure global test runner settings.
```

Comment:
this will probably be very confusing when running multiple test files since it is will be required to be used in each file separately.
I don't have a better solution for this tough

Reply:
My assumption was that people would use a preload to configure global settings like this. That way it will work if we implement single process isolation in the future too. This is also similar to how Jest supports this kind of thing.

Reply:
I assume that as well, but I think it should be clarified in some way that this only affects the tests in the current process. 

I can imagine people expecting this to work

```js
import { run, config } from 'node:test';
import { tap } from 'node:test/reporters';

config.setDefaultSnapshotSerializers([
  (value) => { return JSONStringify(value, null, 2); },
])
run({ files: [path.resolve('./tests/test.js')] })
 .compose(tap)
 .pipe(process.stdout);
```

Reply:
Good point. I'll add more docs.